lib/fika/cli.ex

```diff
@@ -1,10 +1,17 @@
 defmodule Fika.Cli do
+  use Bakeware.Script
+
   require Logger
 
+  @impl Bakeware.Script
   def main(args) do
     parse_args(args)
   end
 
+  defp parse_args([]) do
+    :timer.sleep(:infinity)
```

Comment:
Should we print usage instruction here instead of hanging when no args are passed?